app/services/idv/session.rb

```diff
@@ -3,22 +3,23 @@ class Session
     VALID_SESSION_ATTRIBUTES = %i[
       address_verification_mechanism
       applicant
+      flow_path
```

Comment:
I think we need to set flow_session[:flow_path] as well as idv_session.flow_path for the 50/50 state, because a user might go from a new instance to an old instance. You could add a temporary flow_path= method in Idv::Session that sets both, or add the code back everywhere temporarily. It might be better to add a getter and setter here to make sure we don't miss anything.

Reply:
good point 👍🏿  ... added flow_session["path"] back everywhere temporarily ✅